src/newViews/AnalysisModule/index.tsx

```diff
@@ -115,6 +116,18 @@ const timelineTickSelector = (d: number) => {
 
     return `${year}-${shortMonthNamesMap[month]}`;
 };
+function shouldHideEdit({
+    hasAnalysisFramework,
+    entryPermissions,
+}: {
+    hasAnalysisFramework: boolean;
+    entryPermissions: {
+        create: boolean;
+        modify: boolean;
+    };
+}) {
+    return (!hasAnalysisFramework || !(entryPermissions.create || entryPermissions.modify));
```

Comment:
Why not just use create?

Reply:
We've duplicated the edit entries permissions here